requirements/test.txt

```diff
@@ -4,31 +4,25 @@
 #
 #    pip-compile --output-file requirements/test.txt requirements/base.in requirements/test.in
 #
+-e git+https://github.com/edx/django-celery.git@f87c6f914a1410463f54aebf68458c0653b20602#egg=django-celery==3.2.1+edx.1
 amqp==1.4.9               # via kombu
 anyjson==0.3.3            # via kombu
-appdirs==1.4.3            # via setuptools
 billiard==3.3.0.23        # via celery
 celery==3.1.25
 coverage==4.4.1           # via pytest-cov
 ddt==1.1.1
-django-celery==3.2.1
 django-model-utils==3.0.0
 freezegun==0.3.9
 future==0.16.0
-jsonfield==2.0.1
+jsonfield==2.0.2
 kombu==3.0.37             # via celery
-packaging==16.8           # via setuptools
-py==1.4.33                # via pytest, pytest-catchlog
-pyparsing==2.2.0          # via packaging
+py==1.4.34                # via pytest, pytest-catchlog
 pytest-catchlog==1.2.2
 pytest-cov==2.5.1
 pytest-django==3.1.2
-pytest==3.0.7             # via pytest-catchlog, pytest-cov, pytest-django
-python-dateutil==2.6.0    # via freezegun
+pytest==3.2.0             # via pytest-catchlog, pytest-cov, pytest-django
+python-dateutil==2.6.1    # via freezegun
 python-memcached==1.58
 pytz==2017.2              # via celery, django
-six==1.10.0               # via freezegun, packaging, python-dateutil, python-memcached, setuptools
-
-# The following packages are considered to be unsafe in a requirements file:
-# setuptools                # via pytest
+six==1.10.0               # via freezegun, python-dateutil, python-memcached
 mock
```

Comment:
Yes - the test-extra.txt file:
https://github.com/edx/edx-celeryutils/blob/master/requirements/test-extra.txt
which is added in here:
https://github.com/edx/edx-celeryutils/blob/master/Makefile#L47-L51
Thoughts?